FRAMEWORKS.yml

```diff
@@ -323,3 +323,8 @@ c:
     website: github.com/ohler55/agoo-c
     version: "0.4"
     language: "11"
+kotlin:
+  ktor:
+    website: ktor.io
+    version: 1.1.5
```

Comment:
Could you use only `X.Y` ? I mean `1.1`, this is more readable for audience

Reply:
9faf6c1